learningequality/studio
# Webpack Devserver and Hot Reload

## Description

* Makes our dev build use webpack devserver.
* Adds Hot Module Reloading as an option, using `yarn run devserver:hot`

#### Issue Addressed (if applicable)

Fixes #1444 

## Steps to Test

- Run `yarn run devserver` make sure it works
- Run `yarn run devserver:hot` make sure that when you update JS or CSS/LESS the change shows in the browser without a refresh.

Main question is whether the default full page refresh on `devserver` is to aggressive?

## Checklist


## Comments

*Any additional notes you'd like to add*

## Reviewers

If you are looking to assign a reviewer, here are some options:
- Jordan jayoshih (full stack)
- Aron aronasorman (back end, devops)
- Micah micahscopes (full stack)
- Kevin kollivier (back end)
- Ivan ivanistheone ([Ricecooker](https://github.com/learningequality/ricecooker))
- Richard rtibbles (full stack, [Kolibri](https://github.com/learningequality/kolibri))
- Radina @radinamatic (documentation)
